lib/cylc/scheduler.py

```diff
@@ -258,12 +258,8 @@ def start(self):
                 self.shutdown('ERROR: ' + str(exc))
             except Exception as exc2:
                 # In case of exceptions in the shutdown method itself
-                LOG.warning(exc2)
-            if cylc.flags.debug:
-                raise
-            else:
-                sys.exit(1)
-
+                LOG.exception(exc2)
+            raise exc
```

Comment:
Here we have an unknown `Exception`. I think it is best if we re-raise it regardless of debug mode or log level.

Reply:
:+1: